paper.tex

```diff
@@ -984,12 +1002,12 @@ \subsection*{Illustrative examples}
 distribution on the three effective population sizes
 using a uniform prior on each slice independently.
 The result is reasonably accurate, despite only using 10kb of sequence.
-This approach therefore strengthens
-the inherent ability of ARGs to provide us with temporal
-resolution and context to a dataset of genetic variation.
-It also opens up the opportunity for more efficient optimization
-strategies during inference given that each slice of the ARG can be
-processed independently. 
```

Comment:
I don't disagree with these sentences, but they seem distracting and not necessary.